### PR TITLE
TD Ameritrade offers proprietary two-factor auth solution now

### PR DESCRIPTION
**Pull request by jehart**
TD Ameritrade has a two factor auth application:
https://apps.apple.com/us/app/td-ameritrade-authenticator/id1449911854
https://play.google.com/store/apps/details?id=com.tdameritrade.authenticator&hl=en_US

This is not documented on their website but the solution has been available for about a year now in the app stores.

### CODE REVIEW

File: _data/investing.yml [hunk 1]
```diff
@@ -229,6 +229,8 @@ websites:
     img: tdameritrade.png
     tfa:
       - sms
+      - proprietary
+    doc: https://apps.apple.com/us/app/td-ameritrade-authenticator/id1449911854
```
Comment (phallobst) [inline _data/investing.yml]:
```suggestion
```

**Review by phallobst — CHANGES_REQUESTED** [review]
Since the official documentation (https://www.tdameritrade.com/security/online-safety-tips.page, https://www.tdameritrade.com/security/security-procedures.page) only mentions SMS based 2FA, some screenshots of the 2FA setup procedure would be helpful.

Linking to the app store is not really useful, I suppose.

**Comment by jehart** [comment]
I'm good with that, I wasn't sure how to properly show the links given they don't have it on the website

**Comment by Carlgo11** [comment]
"they" as in your company? 😉 
Just send over some screenshots and it should be good to go.

**Comment by Carlgo11** [comment]
@jehart still need those screenshots in order to approve the PR.

**Comment by jehart** [comment]
Here is an image
Screen Shot 2020-05-30 at 11 00 22 AM

**Comment by jehart** [comment]
> "they" as in your company? 😉
> Just send over some screenshots and it should be good to go.

I don't work there anymore.

**Comment by Carlgo11** [comment]
Closing as the source is gone

**Pull request closed without merging**